git_repo/services/ext/gerrit.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+from ..service import register_target, RepositoryService
+
+@register_target('gerrit', 'gerrit')
+class GerritService(RepositoryService):
+    fqdn = "localhost"
+    port = 29418
+    _max_nested_namespaces = None
+
+    def connect(self):
+        pass
+
+    @classmethod
+    def get_auth_token(cls, login, password, prompt=None):
+        return password
+
+    def get_repository(self, namespace, repo):
+        return namespace + '/' + repo
+
+    def is_repository_empty(self, project):
+        return False
+
+    def get_parent_project_url(self, namespace, repo, rw=True):
+        return None
+
+    def format_path(self, repository, namespace=None, rw=False):
+        if namespace:
+            return 'ssh://{}@{}:{}/{}/{}'.format(self._username, self.fqdn, self.port, namespace, repository)
+        else:
+            return 'ssh://{}@{}:{}/{}'.format(self._username, self.fqdn, self.port, repository)
```

Comment:
how is this different from `super().format_path(...)` ?

needs unit tests please ;-)

Reply:
By the way, I think there is more optimal way to do this. I will change this and upload updated version soon